openedx/core/djangoapps/user_api/accounts/toggles.py

```diff
@@ -24,7 +24,10 @@ def should_redirect_to_order_history_microfrontend():
         REDIRECT_TO_ORDER_HISTORY_MICROFRONTEND.is_enabled()
     )
 
-
+def should_show_linked_accounts_tab():
+    return (
+        configuration_helpers.get_value('SHOW_LINKED_ACCOUNTS', True)
```

Comment:
Please include this inside FEATURES

Reply:
@andrey-canon I thougth it but, for example the platform is using `ENABLE_ORDER_HISTORY_MICROFRONTED` not in features.https://github.com/eduNEXT/edx-platform/pull/273/files/1677cfbc2759186a0a21ba6a7e91d4a9edeed2d5#diff-4791db96231b05cb98a7880993279155e418922dd73823367f3839fd941259f4R23
And this is related to 'disable_order_history_tab'
https://github.com/eduNEXT/edx-platform/pull/273/files/1677cfbc2759186a0a21ba6a7e91d4a9edeed2d5#diff-9628a1b16abda17c654b227da43450afa1beb4c4ea1e2dc8b3909aa76471c927R142

Reply:
do we have a document that specify when use FEATURES and when use a normal setting ?

Reply:
No, I don't know, I would ask. But actually, a thing that I have noticed is that the object `FEATURES` has to be specified with all its variables in order to not left empty the other ones. Meanwhile a normal setting you only have to put it alone.

Reply:
@ Andrey I asked and they tell me is not clear. Some tell me that the feature is to add a full functionality and others tell me will be in `FEATURES`. Do you agree we approve and the we put the PR upstream? Then there they can tell us if there is an order to do it and where is the appropriate place to leave it. https://openedx.atlassian.net/wiki/spaces/AC/pages/40862688/Feature+Flags+and+Settings+on+edx-platform